### DISCUSSION TITLE
`Select.prompt` with more than 10 options and without scroll

### DISCUSSION THREAD

**Discussion by juanfra684** [c4spar/cliffy#762] [Q&A] [answered]
Hi, I couldn't find how to show more than 10 options in the terminal without the scroll. Here is an example:
```javascript
import { Select } from "jsr:@cliffy/prompt@1.0.0-rc.7/select";

const selectNumber = await Select.prompt({
  message: "select a number",
  options: [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20 ]
});
```
Any idea?.

**Comment by c4spar** [comment]
@juanfra684 what do you mean by “without the scroll”? Do you want to have pagination instead of scrolling?

**Reply by juanfra684** [reply]
No, I would like to show all the options in the terminal without scroll or pagination. The terminal has plenty of space for that. When you run my example, it only shows 10 numbers.

**Reply by c4spar** [reply, marked as answer]
To show all options you can set the [`maxRows`](https://cliffy.io/docs@v1.0.0-rc.7/prompt/types/select#max-rows) option to `Infinity`.

```ts
import { Select } from "jsr:@cliffy/prompt@1.0.0-rc.7/select";

const selectNumber = await Select.prompt({
  message: "select a number",
  options: [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20 ],
  maxRows: Infinity,
});
```

**Reply by juanfra684** [reply]
:man_facepalming: Thanks! I didn't see that option.
